system/jlib/jsocket.cpp

```diff
@@ -4340,7 +4340,8 @@ class CSocketEpollThread: public CSocketBaseThread
             else
                 dummysock = ::socket(AF_INET, SOCK_STREAM, 0);
             CHECKSOCKRANGE(dummysock);
-            epoll_op(epfd, EPOLL_CTL_ADD, dummysock, (EPOLLIN | EPOLLERR));
+            // added EPOLLIN also because cannot find anywhere OOB is sent
```

Comment:
I will add to the comment.  Note this code is never called because we do not build without _USE_PIPE_FOR_SELECT_TRIGGER.